hid-t150/attributes.c

```diff
@@ -130,8 +136,15 @@ static ssize_t t150_show_range(struct device *dev, struct device_attribute *attr
 	struct t150 *t150 = dev_get_drvdata(dev);
 	unsigned long flags;
 
+	int dev_max_range;
+
+	if(t150->hid_device->product == 0xb677)
```

Comment:
```
t150->hid_device->product == USB_T150_PRODUCT_ID
```

?

Reply:
For some reason I had an error the first time I tried that. Perhaps I had a typo? Updated to use defined constants.